packages/core/src/modules/credentials/protocol/v2/__tests__/v2.ldproof.credentials.propose-offerBBS.test.ts

```diff
@@ -0,0 +1,250 @@
+import type { Agent } from '../../../../../agent/Agent'
+import type { SignCredentialOptions } from '../../../../../modules/vc/models/W3cCredentialServiceOptions'
+import type { ConnectionRecord } from '../../../../connections'
+import type { ServiceAcceptOfferOptions } from '../../../CredentialServiceOptions'
+import type {
+  AcceptProposalOptions,
+  AcceptRequestOptions,
+  ProposeCredentialOptions,
+} from '../../../CredentialsModuleOptions'
+
+import { setupCredentialTests, waitForCredentialRecord } from '../../../../../../tests/helpers'
+import testLogger from '../../../../../../tests/logger'
+import { KeyType } from '../../../../../crypto/KeyType'
+import { DidKey } from '../../../../../modules/dids'
+import { BbsBlsSignature2020Fixtures } from '../../../../../modules/vc/__tests__/fixtures'
+import { DidCommMessageRepository } from '../../../../../storage'
+import { JsonTransformer } from '../../../../../utils/JsonTransformer'
+import { IndyWallet } from '../../../../../wallet/IndyWallet'
+import { W3cCredential } from '../../../../vc/models/credential/W3cCredential'
+import { CredentialProtocolVersion } from '../../../CredentialProtocolVersion'
+import { CredentialState } from '../../../CredentialState'
+import { CredentialExchangeRecord } from '../../../repository/CredentialExchangeRecord'
+import { V2IssueCredentialMessage } from '../messages/V2IssueCredentialMessage'
+import { V2OfferCredentialMessage } from '../messages/V2OfferCredentialMessage'
+
+let faberAgent: Agent
+let aliceAgent: Agent
+let aliceConnection: ConnectionRecord
+let aliceCredentialRecord: CredentialExchangeRecord
+let faberCredentialRecord: CredentialExchangeRecord
+let wallet: IndyWallet
+let issuerDidKey: DidKey
+let didCommMessageRepository: DidCommMessageRepository
+let signCredentialOptions: SignCredentialOptions
+let verificationMethod: string
+const seed = 'testseed000000000000000000000001'
+describe('credentials, BBS+ signature', () => {
```

Comment:
I don't see any checks for the actual credential that is issued. I think we should add more tests adding the actual format specific data (as this is a test specific to jsonld bbs). For all we know we're just exchanging empty json objects here.